docs/whatsnew/1.1.rst

```diff
@@ -0,0 +1,188 @@
+        Helioprojective [label="Helioprojective Cartesian (HPC)"]
+        GeocentricSolarEcliptic [label="Geocentric Solar Ecliptic (GSE)"]
+        ICRS -> HCRS
+        ICRS -> HeliocentricMeanEcliptic
+        HCRS -> ICRS
+        HCRS -> HeliographicStonyhurst
+        HeliocentricMeanEcliptic -> ICRS
+        HeliocentricMeanEcliptic -> HeliocentricEarthEcliptic
+        HeliocentricMeanEcliptic -> GeocentricEarthEquatorial
+        HeliographicStonyhurst -> HeliographicCarrington
+        HeliographicStonyhurst -> Heliocentric
+        HeliographicStonyhurst -> HCRS
+        HeliographicStonyhurst -> HeliocentricInertial
+        HeliographicCarrington -> HeliographicStonyhurst
+        Heliocentric -> Helioprojective
+        Heliocentric -> HeliographicStonyhurst
+        Helioprojective -> Heliocentric
+        HeliocentricEarthEcliptic -> HeliocentricMeanEcliptic
+        HeliocentricEarthEcliptic -> GeocentricSolarEcliptic
+        GeocentricSolarEcliptic -> HeliocentricEarthEcliptic
+        HeliocentricInertial -> HeliographicStonyhurst
+        GeocentricEarthEquatorial -> HeliocentricMeanEcliptic
+        subgraph cluster_astropy {
+                color=blue
+                fontcolor=blue
+                penwidth=2
+                label=<<b>Frames implemented in Astropy</b>>
+                ICRS
+                HCRS
+                HeliocentricMeanEcliptic
+                astropy [label="Other Astropy frames" shape=box3d style=filled]
+                geocentric [label="Earth-centered frames (including GEO)" shape=box3d style=filled]
+                astropy -> ICRS
+                geocentric -> ICRS
+                ICRS -> astropy
+                ICRS -> geocentric
+        }
+        subgraph cluster_sunpy {
+                color=crimson
+                fontcolor=crimson
+                penwidth=2
+                label=<<b>Frames implemented in SunPy</b>>
+                Helioprojective
+                Heliocentric
+                HeliographicStonyhurst
+                HeliographicCarrington
+                subgraph cluster_sunpy11 {
+                        color=chocolate
+                        fontcolor=chocolate
+                        label=<<b>Requires SunPy 1.1</b>>
+                        HeliocentricInertial
+                        HeliocentricEarthEcliptic
+                        GeocentricSolarEcliptic
+                        GeocentricEarthEquatorial
+                }
+        }
+        newrank=true
+   }
+
+.. _whatsnew-1.1-dl_manager:
+
+Manager for Versioned Data Files
+================================
+
+SunPy 1.1 provides a data manager for versioning and caching remote files.
+The objective of this is provide a way for data required for functions, such as instrument correction routines, to depend on non-local data in a reliable way.
+While also providing a guarantee that a specific version of the code uses a specific data file, with the ability for users to specify updated files.
```

Comment:
This isn't a complete sentence. Is this meant to be combined with the previous sentence?